src/suites/cts/validation/setBindGroup.spec.ts

```diff
@@ -0,0 +1,193 @@
+export const description = `
+setBindGroup validation tests.
+`;
+
+import { TestGroup } from '../../../framework/index.js';
+import GLSL from '../../../tools/glsl.macro.js';
+
+import { ValidationTest } from './validation_test.js';
+
+export class F extends ValidationTest {
+  setup(): { bindGroup: GPUBindGroup; pipelineLayout: GPUPipelineLayout } {
```

Comment:
SetBindGroup also needs a test where we pass an error bindgroup. And a test where we pass the wrong type?

Reply:
If you mean passing like a GPUBuffer instead of a GPUBindGroup, I think we should do all of the tests for incorrect types in one file later.